scripts/sql/statistik_update_care_provider.sql

```diff
@@ -0,0 +1,71 @@
+USE statistik;
+
+DELIMITER $$
+CREATE PROCEDURE updateCareProviderStatistik()
+
+BEGIN
+    -- Declare variables
+    DECLARE updatedCareProviderId VARCHAR(50);
+    DECLARE errorCode CHAR(5) DEFAULT '00000';
+    DECLARE errorMessage TEXT;
+
+    -- Declare handler
+    DECLARE CONTINUE HANDLER FOR SQLEXCEPTION
+    BEGIN
+        GET DIAGNOSTICS CONDITION 1
+            errorCode = RETURNED_SQLSTATE, errorMessage = MESSAGE_TEXT;
+    END;
+
+    SET updatedCareProviderId = 'SE2321000016-39KJ';
+
+    -- Start transaction
+    START TRANSACTION;
+
+
+    DROP TEMPORARY TABLE IF EXISTS originalCareProviderIds;
+    CREATE TEMPORARY TABLE originalCareProviderIds(id VARCHAR(50));
```

Comment:
Set the collation to same as the database. This can be seen in other delete-scripts:

COLLATE utf8mb3_general_ci, -- Set the collate to same as the tables otherwise a table-scan will be made.

See: https://github.com/sklintyg/devops/blob/release/2021-2/scripts/sql/webcert_delete_certificates_by_id.sql